climakitae/threshold_tools.py

```diff
@@ -137,69 +135,32 @@ def get_ks_stat(ams, distr = "gev", multiple_points = True):
         ams = ams.stack(allpoints = ["y", "x"]).squeeze().groupby("allpoints")
 
     def ks_stat(ams):
+        lmoments, fitted_distr = get_fitted_distr(ams, distr, lmom_distr)
+
         if distr == "gev":
-            try:
-                lmoments, fitted_distr = get_fitted_distr(ams, distr, lmom_distr)
-                ks = stats.kstest(
-                    ams,
-                    "genextreme",
-                    args = (lmoments["c"], lmoments["loc"], lmoments["scale"])
-                )
-                d_statistic = ks[0]
-                p_value = ks[1]
-            except (ValueError, ZeroDivisionError):
-                d_statistic = np.nan
-                p_value = np.nan
+            cdf = "genextreme"
+            args = (lmoments["c"], lmoments["loc"], lmoments["scale"])
         elif distr == "gumbel":
-            try:
-                lmoments, fitted_distr = get_fitted_distr(ams, distr, lmom_distr)
-                ks = stats.kstest(
-                    ams, "gumbel_r", args = (lmoments["loc"], lmoments["scale"])
-                )
-                d_statistic = ks[0]
-                p_value = ks[1]
-            except (ValueError, ZeroDivisionError):
-                d_statistic = np.nan
-                p_value = np.nan
+            cdf = "gumbel_r"
+            args = (lmoments["loc"], lmoments["scale"])
         elif distr == "weibull":
```

Comment:
I would propose we add a card to the backlog to handle either weibull min or max behavior as a user option.  I agree that wiebull_min is the default/expected behavior and this is fine for now

Reply:
great, I just made a card!